db/compaction_picker.h

```diff
@@ -84,15 +84,15 @@ class CompactionPicker {
   void ReleaseCompactionFiles(Compaction* c, Status status);
 
   // Returns true if any one of the specified files are being compacted
-  bool FilesInCompaction(const std::vector<FileMetaData*>& files);
+  bool IfFilesInCompaction(const std::vector<FileMetaData*>& files);
```

Comment:
I like Is/Are for this case. Some of the other bool-returning functions in this file are also returning other results in their arguments, so I guess those ones don't need to follow the same convention as functions that only return bool.